stats/test/handler.test.js

```diff
@@ -38,7 +38,7 @@ describe('HTTP request handler', () => {
     server = http.createServer(handler)
     server.listen()
     await once(server, 'listening')
-    baseUrl = `http://127.0.0.1:${server.address().port}`
+    baseUrl = `http://127.0.0.1:${getPort(server)}`
```

Comment:
could we instead do this?
```suggestion
    baseUrl = `http://127.0.0.1:${server.address()!.port}`
```

Reply:
Unfortunately not. 

`server.address()` could be either a string (when listening on a named pipe(?)) or an object with a `port` property (when listening over TCP/IP).

We know that `address` will always be an object, but TypeScript cannot infer that.

It's annoying.